packages/fxa-auth-server/lib/routes/account.ts

```diff
@@ -1449,7 +1449,12 @@ export class AccountHandler {
   async status(request: AuthRequest) {
     const sessionToken = request.auth.credentials;
     if (sessionToken) {
-      return { exists: true, locale: sessionToken.locale };
+      const account = await this.db.account(sessionToken.uid as string);
```

Comment:
Might still be good to have a try/catch on this?

Reply:
Interestingly we don't have `try`/`catch` around some other calls like this. I agree we should though, will add!